### ISSUE TITLE
Support PDF

### ISSUE THREAD

**Issue by typoworx-de** [ZUGFeRD/quba-viewer#1]
Is there any linux-release?

I've been trying to follow manual electron-build instructions (npm install & npm run start) trying to get it working.

This is the stack-trace appearing after trying to open a PDF-File from file-dialog. No matter if using npm or yarn for package-management and also it doesn't make any difference trying to prepend "--unhandled-rejections=strict" after yarn or npm command.

```
Require stack:
- /opt/quba-viewer/main.js
- /opt/quba-viewer/node_modules/electron/dist/resources/default_app.asar/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:972:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:249:921)
    at Module._load (internal/modules/cjs/loader.js:848:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12633)
    at Module.require (internal/modules/cjs/loader.js:1032:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at /opt/quba-viewer/main.js:233:38
(node:8951) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8951) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by jstaerk** [comment]
We wanted to release a linux version with the 0.1.0 but it took just a few extra hours we didn't have so we decided to postpone. I'll let you know once we have a prototype, just yesterday there was another request for a linux version. We currently support UN/CEFACT CII XML, the roadmap is adding UBL/XML format in 0..02 and Factur-X/ZUGFeRD support in 0.3.0, which would also support plain PDFs. Would you be willing to contribute? I could brief you in a video conference (apart from the fact that I'm trying to organize a e-invoicing online barcamp https://forms.gle/dcmR4gaJDDrfmjJh9 ).

**Comment by jstaerk** [comment]
Hi,
as to linux releases: feel free to test http://quba-viewer.org/quba_viewer_0.1.0_amd64.snap and http://quba-viewer.org/quba_viewer-0.1.0.AppImage
kind regards
Jochen

**Comment by jstaerk** [comment]
viewing pdf files is now as well possible in the first developer builds, let me know if you don't want to wait for the final release

**Comment by jstaerk** [comment]
PDF support is now released as part of version 0.5
